package.json

```diff
@@ -164,7 +164,7 @@
     "typescript": "4.8.4"
   },
   "resolutions": {
-    "@kyberswap/ks-sdk-core": "^1.0.2",
+    "@kyberswap/ks-sdk-core": "^1.0.4",
```

Comment:
actually, we dont need this anymore

Reply:
yes we do. This helps force version of core inside of elastic & classic sdk to help them dont break because of mismatch core version